### PR TITLE
Updates readme with usage information

### PR DESCRIPTION
**Pull request by scalvert**
Updates badges to use correct data. Updates usage information.

[Rendered](https://github.com/babel-plugin-ember-test-metadata/babel-plugin-ember-test-metadata/blob/571588e26d25b8d780d3f6d67c39890e50da7c64/readme.md)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -2,37 +2,45 @@
...
+# or
 
-TODO Babel plugin installation
+yarn add babel-plugin-ember-test-metadata -D
+```
 
 ## Usage
 
-TODO Plugin usage in an Ember app
+This plugin needs to be configured in your `ember-cli-build.js` file by adding it you babel's plugins array:
```
Comment (glnster) [inline readme.md:28]: *your Babel's

**Review by drewlee — APPROVED** [review]
Pretty 🤩

**Review by glnster — CHANGES_REQUESTED** [review]
nit: Do we not want to keep the Tests section and just update it to
```sh
yarn test
```

**Comment by glnster** [comment]
Thank you for doing this!

File: readme.md (hunk 1 above)
Comment (scalvert) [reply readme.md]: Thanks.

**Review by glnster — APPROVED** [review]

**Comment by scalvert** [comment]
> nit: Do we not want to keep the Tests section and just update it to
> 
> ```shell
> yarn test
> ```

What's will this help with that they can't find out by just reading the package.json? It's not a huge project, so I think it's pretty easy (not to mention conventional) to just use `(yarn|npm) test` when running tests in a project. If it was unconventional, I'd definitely agree we should add this hint.

**Pull request merged**